doc(release) Release notes collect 24.10 jan26 (#5003)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-24.10/releases/centreon-os.mdx

## Centreon Collect

### 24.10.15

Release date: `January 27, 2025`

<details>
  <summary>Bug fixes</summary>

> - [Broker] Fixed an issue on deleting membership of service to service group.

Reviewer: ```suggestion
- [Broker] Fixed an issue with Broker that resulted in the message "could not delete membership of service to service group".
```

> - [Config] Update logrotate configuration to match packaging permissions.

Reviewer: ```suggestion
- [Config] Updated logrotate configuration to match packaging permissions.
```

> - Fixed an issue where output was truncated for some CMA checks.

Reviewer: ```suggestion
- [CMA] Fixed an issue where output of some CMA checks was truncated.
```

> - [Packaging] Add logrotate to dependency.

Reviewer: ```suggestion
- [Packaging] Added logrotate dependency for Debian.
```

## Centreon Gorgone

### 24.10.10

Release date: `January 27, 2025`

<details>
  <summary>Bug fixes</summary>

- [Config] Update logrotate configuration to match packaging permissions.

> - [Packaging] Add logrotate to dependency.

Reviewer: ```suggestion
- [Config] Updated logrotate configuration to match packaging permissions.
- [Packaging] Added logrotate dependency for Debian.
```
